### PR TITLE
fix namespace in kubernetes yaml cluster

### PR DESCRIPTION
**Pull request by ddorian**
Fixes https://github.com/yugabyte/yugabyte-db/issues/9977

### CODE REVIEW

File: src/components/snippets/kubernetesCode.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 export const yamlDbServerCode = (version) => {
   if (version !== 'latest') {
     return `minikube start --memory=8192 --cpus=4 --disk-size=40g --vm-driver=virtualbox
-curl  https://raw.githubusercontent.com/yugabyte/yugabyte-db/master/cloud/kubernetes/yugabyte-statefulset-rf-1.yaml | sed 's/image: yugabytedb\\/yugabyte\\:latest/image: yugabytedb\\/yugabyte:${version}/g' | kubectl  apply -f -`
+kubectl create namespace yb-demo
+curl  https://raw.githubusercontent.com/yugabyte/yugabyte-db/master/cloud/kubernetes/yugabyte-statefulset-rf-1.yaml | sed 's/image: yugabytedb\\/yugabyte\\:latest/image: yugabytedb\\/yugabyte:${version}/g' | kubectl  apply --namespace='yb-demo' -f -`
```
Comment (polarweasel) [inline src/components/snippets/kubernetesCode.js:5]:
```suggestion
    kubectl create namespace yb-demo
    curl  https://raw.githubusercontent.com/yugabyte/yugabyte-db/master/cloud/kubernetes/yugabyte-statefulset-rf-1.yaml | sed 's/image: yugabytedb\\/yugabyte\\:latest/image: yugabytedb\\/yugabyte:${version}/g' | kubectl  apply --namespace='yb-demo' -f -`
```

Comment (polarweasel) [reply src/components/snippets/kubernetesCode.js:5]: Will this still work correctly with indentation? It's jarring to see without :)

File: src/components/snippets/kubernetesCode.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
   } else {
     return `minikube start --memory=8192 --cpus=4 --disk-size=40g --vm-driver=virtualbox
-curl  https://raw.githubusercontent.com/yugabyte/yugabyte-db/master/cloud/kubernetes/yugabyte-statefulset-rf-1.yaml | kubectl apply -f -`
+kubectl create namespace yb-demo
+curl  https://raw.githubusercontent.com/yugabyte/yugabyte-db/master/cloud/kubernetes/yugabyte-statefulset-rf-1.yaml | kubectl apply --namespace='yb-demo' -f -`
```
Comment (polarweasel) [inline src/components/snippets/kubernetesCode.js:9]: same question as above
```suggestion
    kubectl create namespace yb-demo
    curl  https://raw.githubusercontent.com/yugabyte/yugabyte-db/master/cloud/kubernetes/yugabyte-statefulset-rf-1.yaml | kubectl apply --namespace='yb-demo' -f -`
```

**Review by polarweasel — COMMENTED** [review]
If you can indent in the two if statement blocks, it'd look nicer. Otherwise, LGTM. I assume your code change works. ;-)

File: src/components/snippets/kubernetesCode.js (hunk 1 above)
Comment (yb-andrew) [reply src/components/snippets/kubernetesCode.js:5]: @polarweasel It will work, but the copied text will have the indentation as well.

**Review by yb-andrew — APPROVED** [review]
From a code perspective, this PR LGTM. Regarding the code snippet syntax, I'll defer to Alex's judgement.

**Comment by polarweasel** [comment]
Merging without indentation changes... Don't want to make weird gaps in the copied code!

File: src/components/snippets/kubernetesCode.js (hunk 1 above)
Comment (ddorian) [reply src/components/snippets/kubernetesCode.js:5]: It looked fine on the live preview.

**Comment by ddorian** [comment]
I could replicate the results that the reporting user had and the fix worked for me in ubuntu 18.04.

**Pull request merged**